internal/storage/storage_handle_test.go

```diff
@@ -23,202 +23,200 @@ import (
 
 	mountpkg "github.com/googlecloudplatform/gcsfuse/v2/internal/mount"
 	"github.com/googlecloudplatform/gcsfuse/v2/internal/storage/storageutil"
-	"github.com/jacobsa/oglematchers"
 	. "github.com/jacobsa/ogletest"
+	"github.com/stretchr/testify/suite"
 )
 
 const invalidBucketName string = "will-not-be-present-in-fake-server"
 const projectID string = "valid-project-id"
 
-func TestStorageHandle(t *testing.T) { RunTests(t) }
-
 type StorageHandleTest struct {
+	suite.Suite
 	fakeStorage FakeStorage
 }
 
-var _ SetUpInterface = &StorageHandleTest{}
-var _ TearDownInterface = &StorageHandleTest{}
-
-func init() { RegisterTestSuite(&StorageHandleTest{}) }
+func TestStorageHandleSuite(t *testing.T) {
+	suite.Run(t, new(StorageHandleTest))
+}
 
-func (t *StorageHandleTest) SetUp(_ *TestInfo) {
+func (suite *StorageHandleTest) SetupSuite() {
 	var err error
-	t.fakeStorage = NewFakeStorage()
-	AssertEq(nil, err)
+	suite.fakeStorage = NewFakeStorage()
+	suite.Nil(err)
 }
 
-func (t *StorageHandleTest) TearDown() {
-	t.fakeStorage.ShutDown()
+func (suite *StorageHandleTest) TearDownSuite() {
+	suite.fakeStorage.ShutDown()
 }
 
-func (t *StorageHandleTest) invokeAndVerifyStorageHandle(sc storageutil.StorageClientConfig) {
```

Comment:
this method is not necessary, it avoids duplication but while writing readable unit tests its better to have assertions directly in the test method, its okay to have repetitive code account of readability, see for [ref](https://stackoverflow.com/questions/6453235/what-does-damp-not-dry-mean-when-talking-about-unit-tests)